pom.xml

```diff
@@ -18,6 +18,12 @@
 	<properties>
```

Comment:
Reorder definitions (project and maven before dependencies).
Separate each property that is used for dependencies by a blank line to avod conflicts when creating combined updates.